source/notation.rst

```diff
@@ -17,3 +17,4 @@ Throughout these notes, we rely on different formatting of text to provide seman
     ``<some variable>``, "You know the characters :math:`<` and :math:`>` from mathematical notation where they mean less than and greater than respectively. In this usage, we call them 'angle brackets' and they surround a variable that the user (you) needs to fill in."
     :math:`a`, Denotes a variable in the mathematical sense.
 
+.. note:: All code has been executed using Jupyter. By default when you run a cell within Jupyter, it will display the last variable or unassigned output of a statement without the use of a print statement. This is not a standard function of all IDEs and may lead to differing displays.
```

Comment:
Awesome, and thanks for this! However... 

If you start the standard Python interactive interpreter, then only the last unassigned output is displayed too. A possible difference between these two REPL's is that you can have multiple lines "staged" before execution in IPython/Jupyter. I have never tried to do this in Python, but it may be possible there too. That needs to be checked and this edit updated before merging the PR.

Reply:
I was partly wrong! Here's a test I did. It shows the standard python REPL does not allow staging of multiple commands (as I expected), but **it does** display all `repr()` (note the `...` is a command continuation prompt).

```python
>>> a = 2
>>> for i in range(a):
...   i
... 
0
1
```
In terms of the edit, I think it should highlight the difference.